### PR TITLE
Tailwind Config / Sign In

### PR DESCRIPTION
**Pull request by collin-robinson**
Included Tailwind form style in the sign in form. Added @tailwindcss/forms as dependency and updated @types/react
Created some LL color schemes in Tailwind to extend the theme. (llorange, llyellow) -500 is the logo color.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ FROM node:19-alpine AS base
 # Install dependencies, configure user and group
 RUN apk add --no-cache libc6-compat \
   && addgroup -g 1001 -S nodejs \
-  && adduser -S nextjs -u 1001 \
+  && adduser -S nextjs -u 1001
```
Comment (pid1) [inline Dockerfile:11]: Might need to rebase off dev

File: package.json [hunk 2]
```diff
@@ -12,8 +12,9 @@
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
     "@mui/material": "^5.11.15",
+    "@tailwindcss/forms": "^0.5.3",
```
Comment (pid1) [inline package.json:15]: As discussed, this is fine. No formal process here yet.

**Review by pid1 — APPROVED** [review]
LGTM

**Review by mattkuznicki-ll — APPROVED** [review]
LGTM as well. I'd love to see some of the styling that can be common pulled out into more common styling rather than inline - this can be later in a different PR.

**Pull request merged**